releng/org.eclipse.cdt.repo/category.xml

```
@@ -186,7 +186,7 @@
    <bundle id="org.eclipse.cdt.remote.core.source" version="0.0.0"/>
    <bundle id="com.sun.xml.bind" version="0.0.0"/>
    <bundle id="com.google.gson" version="0.0.0"/>
-   <bundle id="org.freemarker" version="0.0.0"/>
+   <bundle id="org.freemarker.freemarker" version="2.3.32"/>
```

For some reason this silently fails. Checking if a different ID is needed